plugins/tiddlywiki/tiddlyweb/tiddlywebadaptor.js

```diff
@@ -76,6 +76,14 @@ TiddlyWebAdaptor.prototype.getStatus = function(callback) {
 			if(err) {
 				return callback(err);
 			}
+
+			//If Browser-Storage plugin is present, add pre-loaded tiddlers to the event queue to sync to the server 
+			if($tw.wiki.getTiddlerText("$:/config/BrowserStorage/Enabled") === "yes" && $tw.boot.preloadDirty != null) {
+				$tw.boot.preloadDirty.forEach(function(item){
```

Comment:
We use a relatively ancient dialect of approximately ECMAScript 5.1, and so cannot use `.forEach`. We have an alternative utility method:

```
$tw.utils.each($tw.boot.preloadDirty,function(item) {
...
```